types/next-auth.d.ts

```diff
@@ -0,0 +1,17 @@
+import NextAuth from 'next-auth'; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Just wondering where this file came from/what it's used for?

Reply:
this is from the docs, I wanted to remove it but typescript complains if it's not there for some reason 
https://next-auth.js.org/getting-started/typescript#:~:text=ts-,import%20nextauth%20from%20%22next-auth%22,-declare

Reply:
Got it!